docs/rancher/resource-quota.md

```diff
@@ -0,0 +1,62 @@
+
+You can configure the **Namespace** limits as bleow: 
+
+1. Find the newly created project, and click **Create Namespace**.
+1. Specify the desired namespace **Name**, and adjust the limits.
+1. Complete the process by clicking on the **Create** button.
+  ![](/img/v1.2/rancher/create-namespace.png)
+
+## Overhead memory of virtual machine
+Upon creating a virtual machine, the virtual machine controller seamlessly incorporates overhead resources to the virtual machine's configuration. These additional resources are intended to guarantee the consistent and uninterrupted functioning of the virtual machine. It's important to note that configuring memory limits necessitates a higher memory reservation due to the inclusion of these overhead resources.
+
+For more information, see [Memory Overhead](https://kubevirt.io/user-guide/virtual_machines/virtual_hardware/#memory-overhead).
+
+For more information on how the memory overhead is calculated in Kubevirt, check the code [here](https://github.com/kubevirt/kubevirt/blob/v0.54.0/pkg/virt-controller/services/template.go#L1804).
+
+
+
+## Adjust ResourceQuota automatically during the migration
+When the allocated resource quota governed by the `ResourceQuota` object reaches its limit, migrating a virtual machine becomes unfeasible. The migration process involves the automatic creation of a new pod mirroring the resource requirements of the source virtual machine. If these pod creation prerequisites surpass the defined quota, the migration operation cannot proceed.
+
+```suggestion
+_Available as of v1.2.0_
+
+In Harvester, the `ResourceQuota` values are dynamically expanded ahead of migration to accommodate the resource needs of the target virtual machine. Subsequent to migration, the ResourceQuotas are reinstated to their prior configurations.
+
+Please be aware of the following considerations of the automatic resizing of `ResourceQuota`:
+
+- During the process of raising the `ResourceQuota` value, if you create, start, or restore other VMs, Harvester will verify if the resources are sufficient based on the original `ResourceQuota`. If the conditions are not met, the system will alerts that the migration process is not feasible.
+- `ResourceQuota` cannot be changed during VM migration.
+- After expanding `ResourceQuota`, potential resource contention may emerge between non-VM pods and VM pods, leading to migration failures. Consequently, deploying non-VM pods to the same namespace is not recommended.
+- Due to the concurrent limitation of the webhook validator, the VM controller will execute a secondary validation to confirm resource sufficiency. If the resources are insufficient, it will auto config VM's`RunStrategy` to **Halted**, and a new annotation `harvesterhci.io/insufficient-resource-quota: xxxxx` will be added to the VM object, informing you that the VM was shut down due to insufficient resources.
+  ![](/img/v1.2/rancher/vm-annotation-insufficient-resource-quota.png)
```

Comment:
```suggestion
In Harvester, the `ResourceQuota` values dynamically expand ahead of migration to accommodate the resource needs of the target virtual machine. After migration, the ResourceQuotas are reinstated to their prior configurations.

Please be aware of the following considerations of the automatic resizing of `ResourceQuota`:
- When raising the `ResourceQuota` value, if you create, start, or restore other VMs, Harvester will verify if the resources are sufficient based on the original `ResourceQuota`. If the conditions are not met, the system will alert that the migration process is not feasible.
- `ResourceQuota` cannot be changed during VM migration.
- After expanding `ResourceQuota`, potential resource contention may emerge between non-VM pods and VM pods, leading to migration failures. Consequently, deploying non-VM pods to the same namespace is not recommended.
- Due to the concurrent limitation of the webhook validator, the VM controller will execute a secondary validation to confirm resource sufficiency. If the resources are insufficient, it will auto config VM's`RunStrategy` to **Halted**, and a new annotation `harvesterhci.io/insufficient-resource-quota: xxxxx` will be added to the VM object, informing you that the VM was shut down due to insufficient resources.
  ![](/img/v1.2/rancher/vm-annotation-insufficient-resource-quota.png)
```